elyra/scheduler/handler.py

```diff
@@ -37,15 +37,10 @@ def post(self, *args, **kwargs):
 
         pipeline = PipelineParser.parse(pipeline_definition)
 
-        if pipeline.export:
-            PipelineProcessorManager.export(pipeline)
-            json_msg = json.dumps({"status": "ok",
-                                   "message": "Pipeline successfully exported"})
-        else:
-            run_url = PipelineProcessorManager.process(pipeline)
-            json_msg = json.dumps({"status": "ok",
-                                   "message": "Pipeline successfully submitted",
-                                   "url": run_url})
+        run_url = PipelineProcessorManager.process(pipeline)
+        json_msg = json.dumps({"status": "ok",
+                               "message": "Pipeline successfully submitted",
```

Comment:
I believe including them is a "best practice" for APIs, even if our front end doesn't use them.

Reply:
I disagree.  Where is it stated as a best practice to return status _in_ the response body?  Message, perhaps, but I would also contend that's the purview of the caller to decide what should be displayed when successfully calling a REST endpoint.

Reply:
I completely miss that `url` was returned in the line below this. The "status"/"message" practice is for cases where there's "nothing" to return and the response body needs something. In this case the response should just have `url`

Reply:
Ah, yeah, these review "windows" make for a tough view sometimes.  I'm all for learning more about REST APIs since I'm relatively new to this topic.  I thought success code 204 (No Content) was meant for the use case you mention, which is typical for things like DELETE requests (although we chose to return the deleted item with 200 (OK). It just really feels redundant to include it in the body when there's a status code already.

((I'm kinda inclined we should use 204 No Content for our deletes as well.  Seems like returning the instance they at least had a "handle" to is also redundant and error-prone.))

Reply:
I believe this is coming on the other pr, which is waiting on this one.